docs/static/releasenotes.asciidoc

```diff
@@ -38,6 +39,83 @@ This section summarizes the changes in the following releases:
 * <<logstash-7-0-0-alpha2,Logstash 7.0.0-alpha2>>
 * <<logstash-7-0-0-alpha1,Logstash 7.0.0-alpha1>>
 
+[[logstash-7-11-0]]
+=== Logstash 7.11.0 Release Notes
+
+==== New features and enhancements
+
+===== Wildcard support in Central Pipeline Management
+
+With {logstash-ref}/configuring-centralized-pipelines.html[Central Pipeline
+Management], users can create {ls} pipelines in {kib}. Release 7.11.0 introduces
+wildcard support, providing users a more dynamic configuration process. Users no
+longer have to list all pipelines manually. Instead, Logstash will automatically
+pick new pipelines that match the wildcard set in
+`xpack.management.pipeline.id`. 
+
+==== Performance improvements and notable issues fixed
+
+**Central Pipeline Management improvements.** 
+
+This release introduces a fix to a long-standing issue for Logstash Central
+Management in Kibana. When a user tried to delete a Logstash pipeline using
+Kibana, the pipeline wasn’t deleted from the registry. The issue prevented users
+from creating a new pipeline with the same name and config string. The issue has
+been resolved: https://github.com/elastic/logstash/issues/12387[[Meta] support
+xpack delete and recreate pipeline with same pipeline ID and config string
+#12387].
+
+**Persistent Queue corruption after newly allocated page**
+
+A Logstash crash or forceful termination could leave queue in an unrecoverable state,
+causing the following error to be logged as the pipeline starts:
+
+```
+[ERROR][org.logstash.execution.AbstractPipelineExt] Logstash failed to create queue.
+org.logstash.ackedqueue.io.MmapPageIOV2$PageIOInvalidVersionException: Expected page version=2 but found version=0
+```
+
+This has been solved in https://github.com/elastic/logstash/pull/12554[#12554] by ensuring the version file
+is persisted.
+
+**Better handling of fatal exceptions**
+
+Under certain circumstances, Logstash would not respect fatal errors such as java.lang.OutOfMemoryError and continue executing
+without processing data. This issue has been addressed in https://github.com/elastic/logstash/pull/12563[#12563].
+As a consequence of this change, Logstash will now terminate with exit codes associated with the kind of fatal error:
+
+* 128 - InternalError (error in the Java Virtual Machine)
+* 127 - OutOfMemoryError
+* 126 - StackOverflowError
+* 125 - UnknownError (unknown error in the Java Virtual Machine)
+* 124 - IOError
+* 123 - LinkageError (related to JRuby/FFI)
+* 120 - Error
```

Comment:
@kares @karenzone looking at the PR where this comes from I don't see a docs change, let's make sure we document these error codes

Reply:
Thanks for the heads up!  I opened a new issue and assigned both@kares and me: https://github.com/elastic/logstash/issues/12645. I look forward to working with you on this, @kares

Reply:
OK, my bad - did not realize this would be something important to be documented right away.